pepper-apis/src/main/java/org/broadinstitute/ddp/util/I18nUtil.java

```diff
@@ -133,9 +133,9 @@ public static Locale resolvePreferredLanguage(@Nullable Locale preferredLocale,
         if (null != preferredLocale) {
             LanguageRange preferred = new LanguageRange(preferredLocale.toLanguageTag(), LanguageRange.MAX_WEIGHT);
 
-            // This is the locale the user has specifically chosen, so insert it
-            // at the top of the list so it's the first MAX_WEIGHT option.
-            priorityList.add(0, preferred);
+            // This is the locale the user has specifically chosen, insert it
+            // at the bottom of the list so it has the lowest priority
+            priorityList.add(preferred);
```

Comment:
This is a change that gives languages in `Accept-Language` header a priority over the language from the user profile.